lib/routes/towardsdatascience/latest.ts

```diff
@@ -0,0 +1,70 @@
+import { Route } from '@/types';
+import ofetch from '@/utils/ofetch';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+import cache from '@/utils/cache';
+
+export const route: Route = {
+    path: '/latest',
+    categories: ['blog'],
+    example: '/latest',
+    radar: [
+        {
+            source: ['towardsdatascience.com/'],
+        },
+    ],
+    name: 'Towards Data Science',
+    maintainers: ['mintyfrankie'],
+    url: 'towardsdatascience.com/latest',
+    handler,
+};
+
+async function handler() {
+    const baseUrl = 'https://towardsdatascience.com/latest';
+    const feedLang = 'en';
+    const feedDescription = 'Latest articles from Towards Data Science';
+
+    const response = await ofetch(baseUrl);
+    const $ = load(response);
+
+    const list = $('div.postArticle')
+        .toArray()
+        .map((item) => {
+            item = $(item);
+
+            const a = item.find('div.postArticle-content a');
+            const title = a.find('h3').text();
+            const originalLink = a.attr('href').split('?')[0];
+            const link = `https://freedium.cfd/${originalLink}`;
+
+            const header = item.find('div.postMetaInline-authorLockup');
+            const author = header.find('a.ds-link').text().replace('Towards Data Science', '').trim();
+            const pubDate = parseDate(header.find('time').attr('datetime'));
+
+            return {
+                title,
+                link,
+                author,
+                pubDate,
+            };
+        });
```

Comment:
```suggestion
    const list = data.payload.posts.map((item) => {
        const title = item.title;
        const link = `https://towardsdatascience.com/${item.uniqueSlug}`;
        const freediumLink = `https://freedium.cfd/https://towardsdatascience.com/${item.uniqueSlug}`;
        const author = data.payload.references.User[item.creatorId].name;
        const pubDate = parseDate(item.createdAt);

        return {
            title,
            link,
            freediumLink,
            author,
            pubDate,
        };
    });
```
I'm not sure how freedium works. You may need to update `freediumLink`

Reply:
Thanks Tony! Always wondering how you found out these API calls; would you mind sharing your method? I depends on console and also package capture tool, but not too familiar with them.

Regarding freedium, your approach is correct. But somehow I am not sure if the field `freediumLink` will be recognized by RSS readers. Users can open links to full-text content (behind paywall) directly if we put freedium link as `link` itself. What do you think?

Reply:
Btw, if we assign `freediumLink` here, we will also have to change the object got by `cache.tryGet`, because we won't get content with the original links without a subscription, yet with freedium it's very probable.

Reply:
Thanks for the off-topic! Definitely learnt something.